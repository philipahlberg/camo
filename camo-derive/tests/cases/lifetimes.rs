use camo_derive::Camo;

struct Lifetime<'a>(fn(&'a i32));

#[derive(Camo)]
struct Foo<'a> {
    foo: Lifetime<'a>,
}

fn main() {}
