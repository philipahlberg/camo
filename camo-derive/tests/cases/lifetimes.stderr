error: `camo` does not support lifetimes
 --> tests/cases/lifetimes.rs:6:12
  |
6 | struct Foo<'a> {
  |            ^^
